homework/btvn-2/SortLink.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class ListNode
+{
+    public $val = 0;
+    public $next = null;
+
+    function __construct($val)
+    {
+        $this->val = $val;
+    }
+}
+
+class SortLink
+{
+    /**
+     * @param ListNode $head
+     * @return ListNode
+     */
+    function sortList($head) {
+        $linkListToArray = $this->convertLinkListToArray($head);
+
+        rsort($linkListToArray);
+
+        return $this->convertArrayToLinkList($linkListToArray);
+    }
+
+    function convertLinkListToArray($head)
+    {
+        $linkListToArray = [];
+
+        $current = $head;
+
+        do {
+            $linkListToArray[] = $current->val;
+            $current = $current->next;
+        }
+        while($current != null);
+
+        return $linkListToArray;
+    }
+
+    function convertArrayToLinkList($linkListArray)
```

Comment:
this solution is acceptable but consider reimpl using merge sort for the sake of understanding how merge sort work.